ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/base-fee-input/base-fee-input.js

```diff
@@ -105,25 +105,57 @@ const BaseFeeInput = () => {
   const [editingInGwei, setEditingInGwei] = useState(false);
 
   const [maxBaseFeeGWEI, setMaxBaseFeeGWEI] = useState(() => {
-    if (
-      estimateUsed !== PRIORITY_LEVELS.CUSTOM &&
-      advancedGasFeeValues?.maxBaseFee
+    if (estimateUsed !== PRIORITY_LEVELS.CUSTOM) {
+      if (advancedGasFeeValues?.maxBaseFeeGWEI) {
```

Comment:
Instead of checking for `advancedGasFeeValues.?` over and over, is it worth it to wrap this whole thing in an `if (advancedGasFeeValues !== null) {` check?